VEDA/settings/production.py

```diff
@@ -10,7 +10,7 @@
 
 ALLOWED_HOSTS = ['*']
 
-LOGGING = get_logger_config()
+LOGGING = get_logger_config(debug=True)
```

Comment:
@yro please remove this as I've already a PR for this which is going to merged into master. (to avoid conflicts)